### PR TITLE
Demo branch 02 with IN - User Profile #37

### PR DESCRIPTION
**Pull request by jonquag**
Here's an updated look on the UI for the User Profile component. 
image

### CODE REVIEW

File: client/src/pages/layout/Navbar.js [hunk 1]
```diff
@@ -55,46 +54,56 @@ const Navbar = () => {
...
+                                <Badge badgeContent={12} className={classes.badge} />
+                            </ListItem>
                         </List>
                     </div>
                     <div className={classes.profile}>
                         <Avatar
-                            alt="profiel_img"
-                            src={profile.imageUrl.length ? profile.imageUrl : profileImg}
+                            alt="profile_img"
+                            src={profile.imageUrl}
```
Comment (timegambit) [inline client/src/pages/layout/Navbar.js]: If we're using the default icon in DropzonePicture we want to use it here as well

File: client/src/components/DragzonePicture.js [hunk 2]
```diff
@@ -1,75 +1,77 @@
...
+            if (droppedFiles.length) {
+                setUploading(true);
 
-            const form = new FormData();
-            form.append('image', droppedFiles[0]);
-            const res = await axios.post('/upload', form)
-            .catch((err) => console.log(err));
+                const form = new FormData();
+                form.append('image', droppedFiles[0]);
+                // form.append('firstName', profile.firstName);
```
Comment (timegambit) [inline client/src/components/DragzonePicture.js]: Remove commented code

**Comment by jonquag** [comment]
```    
const handleSubmit = e => {
        e.preventDefault();
        const userInfo = {
            firstName,
            lastName,
            email,
            location,
            phone,
        };
        axios.put(`/profile/${profile.user._id}`, userInfo);
        console.log('update changes clicked');
        // setfirstName = '',
        // setlastName= '',
        // setemail = '',
        // setlocation = '',
        // phone = ''.
    };
```
The `<TextField>` Elements are also not able to be adjusted after the `put()` requests are sent. May need some help with this. I tried to set the variables back to empty strings (see commented code) but wasn't having any luck.

File: client/src/components/Profile/UserInfo.js [hunk 3]
```diff
@@ -15,82 +17,137 @@ const useStyles = makeStyles({
...
+                >
+                    <Grid item>
+                        <InputLabel className={classes.labelStyles}>
+                            First Name
+                        </InputLabel>
+                        <TextField
+                            id="outlined"
+                            hinttext="Change first name.."
+                            variant="outlined"
+                            value={profile.firstName}
```
Comment (timegambit) [inline client/src/components/Profile/UserInfo.js]: I don't think this is going to work. The value of profile.firstName never changes when editing, so I believe the TextFields will be uneditable.

Comment (jonquag) [reply client/src/components/Profile/UserInfo.js]: What if we take the values retrieved from the DB and put them into the setFirstName() etc variables?

Comment (timegambit) [reply client/src/components/Profile/UserInfo.js]: I think that would work, and then you'd want to change the value of each TextField to firstName, lastName, etc. and not profile.firstName, profile.lastName, etc.

**Pull request closed without merging**